Source/Dafny/Resolver.cs

```diff
@@ -687,6 +694,16 @@ enum ValuetypeVariety {
 
       Type.DisableScopes();
       CheckDupModuleNames(prog);
+
+      foreach (var module in prog.Modules()) {
+        foreach (var r in rewriters) {
```

Comment:
I'd prefer avoiding letter single variables. Why not write `rewriter` ?

Reply:
I prefer avoiding single letter variables. That was part of the code I just copy-pasted. With the upcoming commit, in every loop of this file that is ` (var r in rewriters)`, I renamed the variable to "rewriter", that sounds better.